src/tests/ftest/util/osa_utils.py

```diff
@@ -115,6 +116,53 @@ def get_pool_version(self):
         data = self.dmg_command.pool_query(self.pool.uuid)
         return int(data["response"]["version"])
 
+    @fail_on(CommandFailure)
+    def get_ipaddr_for_rank(self, rank=None):
```

Comment:
(pylint-inconsistent-return-statements) Either all return statements in a function should return an expression, or none of them should.